File: Q06/main.c
#include<stdio.h>
int main(){
    int n;
    scanf("%d",&n);
    int arr[n][n];

    for(int i=0;i<n;i++){
        for(int j=0;j<n;j++){
            scanf("%d",&arr[i][j]);
        }
    }
    int rd=0,ld=0;

    for(int i=0;i<n;i++){
            rd+=arr[i][i];
            ld+=arr[i][n-i-1];
    }
    printf("%d\n%d",rd,ld);
    return 0;
}

File: Q06/main.py
n = int(input())
arr=[]

for i in range(n):
    arr.append(list(map(int,input().split()))[:n])

ld=rd=0

for i in range(n):
    rd+=arr[i][i]
    ld+=arr[i][n-i-1]

print(rd,ld)